### PR TITLE
More Logging Messages

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

-  More logging for troubleshooting errors.

**Related issue(s)**

### CODE REVIEW

File: tests/fast-integration/utils/index.js [hunk 1]
```diff
@@ -116,7 +121,7 @@ function convertAxiosError(axiosError, message) {
   return new Error(message);
 }
 
-// "polyfill" for Promise.allSettled
+// 'polyfill' for Promise.allSettled
```
Comment (jaroslaw-pieszka) [inline tests/fast-integration/utils/index.js:124]: Why?

**Review by PK85 — APPROVED** [review]

**Pull request merged**